### DISCUSSION TITLE
useLog方法编译后语法好像不太支持

### DISCUSSION THREAD

**Discussion by webdq** [zouyaoji/vue-cesium#217] [Q&A]
"vite": "^2.8.0",
"vue": "^3.2.25",
"vue-cesium": "^3.0.14",

[![jVgong.png](https://s1.ax1x.com/2022/06/27/jVgong.png)](https://imgtu.com/i/jVgong)
[![jVg77j.png](https://s1.ax1x.com/2022/06/27/jVg77j.png)](https://imgtu.com/i/jVg77j)

**Comment by zouyaoji** [comment]
我测试了，没毛病哈。请注意要写到 setup 里面。

**Reply by webdq** [reply]
找到原因了，是我的vite配置里剔除了console.log导致编译打包后显示的错误
